core-services/sshd-bastion/ppc64le-libvirt/OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
+  - stevekuznetsov
```

Comment:
Steve is redundant since he owns everything sshd-bastion. :)

Reply:
how could you call steve redundant @jaypoulz?! :D

(fixing. and am going to resubmit for @manojnkmar since his org approval's still pending)

Reply:
You're going to have to run 'make prow-config' as well

Reply:
@hamzy I did that. Did it not do what it was supposed to do? After it ran (with no errors), nothing new showed up in my local dir. Also can we pick this up in pull #8019?

Reply:
You are still getting the failure: ci/prow/prow-config — Job failed. 
Look at https://prow.svc.ci.openshifta.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/7937/pull-ci-openshift-release-master-prow-config/7444

Reply:
When you run that make command, it will modify files that you have to git add and commit.